integration-tests/api/__tests__/batch-jobs/order/export.js

```diff
@@ -0,0 +1,236 @@
+const path = require("path")
+const fs = require("fs/promises")
+import { sep, resolve } from "path"
+
+const setupServer = require("../../../../helpers/setup-server")
+const { useApi } = require("../../../../helpers/use-api")
+const { initDb, useDb } = require("../../../../helpers/use-db")
+
+const adminSeeder = require("../../../helpers/admin-seeder")
+const userSeeder = require("../../../helpers/user-seeder")
+const orderSeeder = require("../../../helpers/order-seeder")
+
+const adminReqConfig = {
+  headers: {
+    Authorization: "Bearer test_token",
+  },
+}
+
+jest.setTimeout(1000000)
+
+describe("Batchjob with type order-export", () => {
+  let medusaProcess
+  let dbConnection
+  let exportFilePath = ""
+  let topDir = ""
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", "..", ".."))
+    dbConnection = await initDb({ cwd })
+    medusaProcess = await setupServer({
+      cwd,
+      redisUrl: "redis://127.0.0.1:6379",
+      uploadDir: __dirname,
+      verbose: false,
+    })
+  })
+
+  afterAll(async () => {
+    if (topDir !== "") {
+      await fs.rmdir(resolve(__dirname, topDir), { recursive: true })
+    }
+
+    const db = useDb()
+    await db.shutdown()
+
+    medusaProcess.kill()
+  })
+
+  beforeEach(async () => {
+    try {
+      await adminSeeder(dbConnection)
+      await userSeeder(dbConnection)
+      await orderSeeder(dbConnection)
+    } catch (e) {
+      console.log(e)
+      throw e
+    }
+  })
+
+  afterEach(async () => {
+    const db = useDb()
+    await db.teardown()
+
+    const isFileExists = (await fs.stat(exportFilePath))?.isFile()
+
+    if (isFileExists) {
```

Comment:
praise: Very nice that way of managing the files that have been generated, do you think you could also use that approach for the product export test case?

Reply:
ofc. 

I was think regarding to exports, should we by default export to something like `export/{export-filename}` instead of exporting to the root directory. Or should we have everything in the root directory? 

I think it becomes very messy quickly, and at least S3, DO and Minio all support creating directories dynamically like this. We could also take it a step further and use `export/{product|order}/{export-filename}`

wdyt?

Reply:
I think that it is a very nice idea indeed ;)